business/tls.go

```diff
@@ -145,7 +145,7 @@ func (in *TLSService) NamespaceWidemTLSStatus(ctx context.Context, namespace, cl
 	}
 
 	return models.MTLSStatus{
-		Status:          mtlsStatus.NamespaceMtlsStatus(namespace).OverallStatus,
+		Status:          mtlsStatus.NamespaceMtlsStatus(namespace, in.businessLayer.App.conf).OverallStatus,
```

Comment:
Can't just all Services have conf?

Reply:
We've been slowly adding it in over time but having it consistent would be great.

Reply:
I think there is one more place to replace in this class.